### ISSUE TITLE
Performing augmentation and distortions like ImageDataGenerator()

### ISSUE THREAD

**Issue by deshwalmahesh** [jangedoo/age-gender-race-prediction#1]
I was looking for a data generator that similar to ```ImageDataGenerator``` in keras but with muti-output labels. Problem with keras' is that it can take only single output. Could you please help me with that.

**Comment by AGenchev** [comment]
while your issue seems not much related to a problem with this interesting project, I agree that we can add image augmentations and increase training epoches to train better. I see this project as example how multi-output can be done.
I am thankful that it run with very little issues with TensorFlow 2.2
